src/lib/button/button.ts

```diff
@@ -157,6 +157,17 @@ export class MdButton implements OnDestroy {
   _isRippleDisabled() {
     return this.disableRipple || this.disabled;
   }
+
+  /** Gets whether the button has one of the given attributes
+   * with either an 'md-' or 'mat-' prefix.
+   */
+  _hasAttributeWithPrefix(unprefixedAttributeNames: string[]) {
```

Comment:
It just occured to me that this would be more appropriate as a rest argument:
```ts
  _hasAttributeWithPrefix(...unprefixedAttributeNames: string[]) {
    // ...
  }
```
Then the call sites don't need the array syntax.